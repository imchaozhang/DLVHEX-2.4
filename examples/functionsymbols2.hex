p(a).
limit(a).
limit(aa).
p(Y) :- p(X), &idc[f(X)](f(Y)), limit(X). 
