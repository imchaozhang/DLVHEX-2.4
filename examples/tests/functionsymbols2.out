{p(a),limit(a),limit(aa)}
